index.bs

```diff
@@ -9911,7 +9911,37 @@ validity of a given {{AudioWorkletProcessor}} subclass.
 			<em>Return type:</em> {{boolean}}
 		</div>
 </dl>
-	
+
```

Comment:
You should have an introductory sentence here just to make clear the separation between the example and the parameters of the process() method.  Now it just reads funny.

Reply:
Done.